website/src/components/broadcast/MatchScoreboard.vue

```diff
@@ -39,8 +41,23 @@ import { bg } from "@/utils/images";
 
 export default {
     name: "MatchScoreboard",
-    components: { ThemeTransition, Squeezable, ThemeLogo },
-    methods: { themeBackground1, logoBackground1, cleanID, bg },
+    components: {
+        ThemeTransition,
+        Squeezable,
+        ThemeLogo
+    },
+    methods: {
+        themeBackground1,
+        logoBackground1,
+        cleanID,
+        bg,
+        mapTypeIcon(map) {
+            if (map.map) return MapTypeIcons[map.map.type];
+            if (!this.match?.first_to || !this.mapTypes) return null;
```

Comment:
Do we need `!this.match?.first_to` here? Don't think it's relevant to restrict to it.
```suggestion
            if (!this.mapTypes) return null;
```

Reply:
I don't know if it's a useful check, but if there is no `first_to` there is no dummy maps, so for a map that's not selected explicitly we wouldn't have an icon anyway cuz the map don't exist anywhere? I just copied the chekc from https://github.com/slmnio/slmngg-sfc/blob/master/website/src/components/broadcast/MatchScoreboard.vue#L72